jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/MavenSettingsServerCredentials.java

```diff
@@ -108,4 +109,23 @@ public String getPasswordDescriptor() {
           }
         });
   }
+
+  @Nullable
+  Server getServerFromMavenSettings(String registry) {
+    // Find and remove port (if present)
+    if (registry == null) {
+      return null;
+    }
```

Comment:
You don't need this null check. We check nullability with NullAway, and unless the argument is marked with `@Nullable`, it is never null.